src/framework/components/rigid-body/system.js

```diff
@@ -181,6 +181,13 @@ Object.assign(pc, function () {
                 var solver = new Ammo.btSequentialImpulseConstraintSolver();
                 this.dynamicsWorld = new Ammo.btDiscreteDynamicsWorld(dispatcher, overlappingPairCache, solver, collisionConfiguration);
 
+                if (this.dynamicsWorld.setInternalTickCallback) {
+                    var checkForCollisionsPointer = Ammo.addFunction(this._checkForCollisions.bind(this), 'vif');
+                    this.dynamicsWorld.setInternalTickCallback(checkForCollisionsPointer);
+                } else {
+                    console.warn("WARNING: This version of ammo.js can potentially fail to report contacts. Please update it to the latest version.");
```

Comment:
Wrap this with #ifdef DEBUG and #endif. For example: https://github.com/playcanvas/engine/blob/master/src/scene/graph-node.js#L1050

Reply:
If I do that the linter complains about the else statement not being on the same line as the closing bracket of the if statement 😞

Reply:
Linter didn't seem to have a problem with just wrapping the inside of the else. Pushed what should resolve the rest.

Reply:
How about:

```
if (this.dynamicsWorld.setInternalTickCallback) {
    var checkForCollisionsPointer = Ammo.addFunction(this._checkForCollisions.bind(this), 'vif');
    this.dynamicsWorld.setInternalTickCallback(checkForCollisionsPointer);
}

#ifdef DEBUG
if (!this.dynamicsWorld.setInternalTickCallback) {
    console.warn("WARNING: This version of ammo.js can potentially fail to report contacts. Please update it to the latest version.");
}
#endif
```

Reply:
Oh cool, sounds good.